File: korge-spine/src/jvmMain/java/com/badlogic/gdx/graphics/GL20.java
package com.badlogic.gdx.graphics;

public interface GL20 {
    int GL_SRC_ALPHA = 0;
    int GL_ONE = 1;
    int GL_ONE_MINUS_SRC_ALPHA = 2;
    int GL_DST_COLOR = 3;
    int GL_ONE_MINUS_SRC_COLOR = 4;

    int GL_POINTS = 0;
    int GL_LINES = 1;
    int GL_TRIANGLES = 2;
    int GL_BLEND = 0;

    void glEnable(int glBlend);

    void glBlendFunc(int srcFunc, int glOneMinusSrcAlpha);

    void glDepthMask(boolean b);

    void glDisable(int glBlend);

    void glBlendFuncSeparate(int blendSrcFunc, int blendDstFunc, int blendSrcFuncAlpha, int blendDstFuncAlpha);
}

File: korge-spine/src/commonMain/kotlin/com/esotericsoftware/spine/rendering/GL20.kt
package com.esotericsoftware.spine.rendering

interface GL20 {
    fun glEnable(glBlend: Int)
    fun glBlendFunc(srcFunc: Int, glOneMinusSrcAlpha: Int)
    fun glDepthMask(b: Boolean)
    fun glDisable(glBlend: Int)
    fun glBlendFuncSeparate(blendSrcFunc: Int, blendDstFunc: Int, blendSrcFuncAlpha: Int, blendDstFuncAlpha: Int)

    companion object {
        const val GL_SRC_ALPHA = 0
        const val GL_ONE = 1
        const val GL_ONE_MINUS_SRC_ALPHA = 2
        const val GL_DST_COLOR = 3
        const val GL_ONE_MINUS_SRC_COLOR = 4
        const val GL_POINTS = 0
        const val GL_LINES = 1
        const val GL_TRIANGLES = 2
        const val GL_BLEND = 0
    }
}
